shell/edit/workload/mixins/workload.js

```diff
@@ -184,25 +184,17 @@ export default {
           name:            `container-0`,
         }];
 
-        const metadata = { ...this.value.metadata };
-
-        const podSpec = { template: { spec: { containers: podContainers, initContainers: [] }, metadata } };
+        const podSpec = {
+          containers:     podContainers,
+          initContainers: [],
+        };
 
         this.$set(this.value, 'spec', podSpec);
       }
     }
 
-    // EDIT view for POD
-    // Transform it from POD world to workload
-    if ((this.mode === _EDIT || this.mode === _VIEW || this.realMode === _CLONE ) && this.value.type === 'pod') {
-      const podSpec = { ...this.value.spec };
-      const metadata = { ...this.value.metadata };
-
-      this.$set(this.value.spec, 'template', { spec: podSpec, metadata });
-    }
```

Comment:
I don't believe that this is required anymore as it appears to have existed so that we can translate a valid pod spec into an invalid one.